addons/point_of_sale/static/tests/pos/tours/utils/chrome_util.js

```diff
@@ -14,7 +14,13 @@ export function clickMenuButton() {
     };
 }
 export function clickMenuOption(name, options) {
-    return [clickMenuButton(), clickMenuDropdownOption(name, options)];
+    return [waitForMenuButtons(), clickMenuButton(), clickMenuDropdownOption(name, options)];
+}
+export function waitForMenuButtons() {
+    return {
+        content: `Wait for the menu buttons to be available`,
+        trigger: `.pos-rightheader button:has(.fa-bars)`,
```

Comment:
Hello, I randomly went into this PR as I had the issue on one of my branches.

How is adding a wait on the button going to solve the issue?
Normally the click helper is already waiting for the button too, so this feels redundant and not the correct fix.